src/ifp/include/ifp/InitFloorplan.hh

```diff
@@ -111,6 +112,13 @@ class InitFloorplan
                int core_uy,
                int factor,
                int row_index);
+  int makeHybridRows(const std::vector<dbSite*>& hybrid_sites,
```

Comment:
I think we should name this more generically, seems like we should tie it more to the concept of "patterns" rather than hybrid rows. 

"makeRowPattern"
"generateContiguousRowPattern"